### PR TITLE
tools: openwrt: update feed-prpl hash

### PR DESCRIPTION
**Pull request by tomereli**
On normal operation, prplmesh is reconfiguring the hostapd as part of
autoconfiguration where it gets the VAPs configuration from the
controller in the WSC M2 message.
When prplmesh is stopped, the wireless configuration should be
reconfigured based on the UCI settings, which is missing now in the
prplmesh init script.

Update feed-prpl hash to include the following merge request which
runs /etc/init.d/network restart to restart the wireless LAN when prplmesh is
stopped - https://git.prpl.dev/prplmesh/feed-prpl/-/merge_requests/2.

Signed-off-by: Tomer Eliyahu <tomer.b.eliyahu@intel.com>

### CODE REVIEW

File: tools/docker/builder/openwrt/build.sh
Comment (arnout) [inline tools/docker/builder/openwrt/build.sh]: This obviously has to be updated to the actual commit hash after merging !2. So I'm going to add the don't merge tag.

**Review by arnout — APPROVED** [review]

**Comment by tomereli** [comment]
@rmelotte looks like the prplmesh dummy controller answered:

> DEBUG 23:54:12:167 <1984281848> backhaul_manager_thread.cpp[1267] --> sending autoconfig search message, bridge_mac=10:0c:6b:a1:bd:6d
TRACE 23:54:12:168 <1984281848> backhaul_manager_thread.cpp[1026] --> FSM: SEND_AUTOCONFIG_SEARCH_MESSAGE --> WAIT_FOR_AUTOCONFIG_RESPONSE_MESSAGE
DEBUG 23:54:12:996 <1984281848> backhaul_manager_thread.cpp[2798] --> received autoconfig response message
DEBUG 23:54:12:996 <1984281848> backhaul_manager_thread.cpp[2808] --> tlvSupportedRole->value()=0
DEBUG 23:54:12:997 <1984281848> backhaul_manager_thread.cpp[2822] --> prplMesh controller: received ACTION_CONTROL_SLAVE_HANDSHAKE_RESPONSE from 46:55:66:77:00:00
INFO 23:54:12:997 <1984281848> backhaul_manager_thread.cpp[2832] --> update controller_bridge_mac=46:55:66:77:00:00
DEBUG 23:54:12:998 <1984281848> backhaul_manager_thread.cpp[2836] --> tlvSupportedFreqBand->value()=0
DEBUG 23:54:12:998 <1984281848> backhaul_manager_thread.cpp[2855] --> received auto-config reply for 2.4GHz band

File: tools/docker/builder/openwrt/build.sh
Comment (tomereli) [reply tools/docker/builder/openwrt/build.sh]: Actually we're using FF merge so the commit SHA1 wasn't changed.

**Review by rmelotte — APPROVED** [review]

**Pull request merged**
